nifi-nar-bundles/nifi-avro-bundle/nifi-avro-processors/src/main/java/org/apache/nifi/processors/avro/ExtractAvroMetadata.java

```diff
@@ -54,7 +54,7 @@
 import org.apache.nifi.processor.Relationship;
 import org.apache.nifi.processor.exception.ProcessException;
 import org.apache.nifi.processor.io.InputStreamCallback;
-import org.apache.nifi.processor.util.StandardValidators;
+import org.apache.nifi.util.validator.StandardValidators;
```

Comment:
Wouldn't this break all the existing processors that are not part of NiFi distribution? Basically any custom/in-house developed processor?